src/DbUpdateService.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.EntityFrameworkCore;
+using ModelRepoBrowser.Crawler;
+
+namespace ModelRepoBrowser;
+
+/// <summary>
+/// Service that periodically updates the Model Repository Database using <see cref="IRepositoryCrawler"/>.
+/// </summary>
+public class DbUpdateService : BackgroundService
+{
+    private readonly IServiceScopeFactory scopeFactory;
+    private readonly ILogger<DbUpdateService> logger;
+    private readonly IConfiguration configuration;
+
+    public TimeOnly PreferedTime { get; set; }
+
+    public DbUpdateService(IServiceScopeFactory scopeFactory, ILogger<DbUpdateService> logger, IConfiguration configuration)
+    {
+        this.scopeFactory = scopeFactory;
+        this.logger = logger;
+        this.configuration = configuration;
+
+        PreferedTime = new TimeOnly(1, 0);
+    }
+
+    private async Task UpdateModelRepoDatabase()
+    {
+        logger.LogInformation("Updating ModelRepoDatabase...");
+
+        if (!Uri.TryCreate(configuration["Crawler:RootRepositoryUri"], UriKind.RelativeOrAbsolute, out var rootUri))
+        {
+            logger.LogError("Unable to parse configuration Crawler:RootRepositoryUri. Database update skipped.");
+            return;
+        }
+
+        try
+        {
+            using (var scope = scopeFactory.CreateScope())
+            {
+                var crawler = scope.ServiceProvider.GetRequiredService<IRepositoryCrawler>();
+                var repositories = await crawler.CrawlModelRepositories(rootUri).ConfigureAwait(false);
+
+                var context = scope.ServiceProvider.GetRequiredService<RepoBrowserContext>();
```

Comment:
REQ: `context` nach dem Update disposen
```suggestion
                using var context = scope.ServiceProvider.GetRequiredService<RepoBrowserContext>();
```